src/tools/telephony/attended_transfer.py

```
@@ -139,69 +149,54 @@ async def execute(self, parameters: Dict[str, Any], context: ToolExecutionContex
+                dial_timeout_sec=dial_timeout_sec,
+                moh_class=moh_class,
+                screening_max_seconds=caller_screening_max_seconds,
+                screening_silence_ms=caller_screening_silence_ms,
             )
-        except Exception:
-            result = None
-            logger.error("Failed to originate attended transfer agent leg", call_id=call_id, exc_info=True)
+            if engine and hasattr(engine, "_fire_and_forget_for_call"):
+                engine._fire_and_forget_for_call(call_id, workflow, name=f"attx-screening-{call_id}")
+            else:
+                asyncio.create_task(workflow)
```

Ruff (0.15.7)

[warning] Store a reference to the return value of `asyncio.create_task`

(RUF006)